Sources/TuistSupport/Extensions/AbsolutePath+Extras.swift

```diff
@@ -67,3 +87,18 @@ extension AbsolutePath: ExpressibleByStringLiteral {
         self = AbsolutePath(value)
     }
 }
+
+extension String {
+    private var isGlobComponent: Bool {
+        self == "**" || self == "*" || hasPrefix("*.")
```

Comment:
I might be wrong, but I think patterns like the following are also glob components:

```
Sources/*Tests.swift
```